mysql-test/include/has_tokudb.inc

```diff
@@ -0,0 +1,18 @@
+#
```

Comment:
I agree and had a bit of a hard time myself coming up with a name for it. Has seemed logical because of the precedent set by 'have' meaning "must have or fail". One of the two most difficult issues in programming :)